deps/LLVMExtra/lib/llvm-api.cpp

```diff
@@ -561,3 +566,13 @@ LLVMValueRef LLVMConstDataArray(LLVMTypeRef ElementTy, const void *Data, unsigne
     StringRef S((const char *)Data, NumElements * unwrap(ElementTy)->getPrimitiveSizeInBits() / 8);
     return wrap(ConstantDataArray::getRaw(S, NumElements, unwrap(ElementTy)));
 }
+
+LLVMTypeRef LLVMGetType(LLVMValueRef V) {
+    return wrap(unwrap(V)->getType());
```

Comment:
What's wrong with `LLVMTypeOf`? That's already wrapped as `llvmtype`.

Reply:
Nothing, I just missed it :laughing: